packages/components/src/dropdown/stories/index.js

```diff
@@ -1,81 +1,72 @@
-/**
- * WordPress dependencies
- */
-import {
-	more,
-	arrowLeft,
-	arrowRight,
-	arrowUp,
-	arrowDown,
-} from '@wordpress/icons';
```

Comment:
I had to do some basic cleanup of the story first. It may be easier to review the commits separately.